coding_systems/versioning/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+from opencodelists.hash_utils import hash, unhash
+
+
+class CodingSystemVersionManager(models.Manager):
+    def get_by_hash(self, hash):
+        """Return the CodingSystemVersion with given hash."""
+        id = unhash(hash, "CodingSystemVersion")
+        return self.get(id=id)
+
+
+class CodingSystemVersion(models.Model):
+    coding_system = models.CharField(max_length=10)
+    version = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
I think I had an idea initially that  this would be an official version identifier and they might not all have one, but I guess even if they didn't, we should label them with one anyway

Reply:
> we should label them with one anyway

I agree.